Dockerfile

```
@@ -0,0 +1,9 @@
+FROM node:latest
+
+RUN mkdir /app
+WORKDIR /app
+ADD package.json /app/
+COPY . /app/
+RUN npm install
```

```suggestion
RUN npm ci --silent
```

it will install the exact same version as in the lock file